src/Selenium2Driver.php

```diff
@@ -214,6 +214,23 @@ public static function getDefaultCapabilities()
         );
     }
 
+    /**
+     * Checks if the WebDriver session is W3C compatible.
+     *
+     * @return bool
+     *
+     * @throws UnknownCommand
+     */
+    public function isW3C() {
+        try {
+            // This method doesn't exist in older versions of php-webdriver.
```

Comment:
I'd rather check with `method_exists` here rather than relying on the undefined method call triggering an UnknownCommand exception.